frontend/src/app/main/data/workspace/persistence.cljs

```diff
@@ -300,8 +300,7 @@
                   (rx/mapcat
                     #(rx/zip (rp/query :file-library {:file-id library-id})
                              (rp/query :media-objects {:file-id library-id
-                                                       :is-local false})
-                             (rp/query :colors {:file-id library-id}))))
+                                                       :is-local false}))))
```

Comment:
Why removing colors? Shouldn't we instead add typographies?

Reply:
No, that query is not necessary anymore because the color data comes inside the file.